graphql2/graphqlapp/alert.go

```diff
@@ -75,8 +76,29 @@ func (a *AlertLogEntry) notificationSentState(ctx context.Context, obj *alertlog
 		status = "OK"
 	}
 
+	var prefix string
+	switch s.State {
+	case notification.MessageStatePending:
+		prefix = "Pending"
+	case notification.MessageStateSending:
+		prefix = "Sending"
+	case notification.MessageStateSent:
+		prefix = "Sent"
+	case notification.MessageStateDelivered:
+		prefix = "Delivered"
+	case notification.MessageStateFailedTemp:
+		prefix = "Failed"
+	case notification.MessageStateFailedPerm:
+		prefix = "Failed"
```

Comment:
These can be combined in Go:
```suggestion
	case notification.MessageStateFailedTemp, notification.MessageStateFailedPerm:
		prefix = "Failed"
```